src/west/manifest-schema.yml

```diff
@@ -130,3 +130,31 @@ mapping:
           clone-depth:
             required: false
             type: int
+
+  # The "self" key specifies values for the project containing the manifest
+  # file.
+  #
+  # When using West to initialize a project there must be a repository
+  # containing a manifest files with sub-projects.
+  # In case the Git repository containing the manifest file must be located in
+  # The path field can be used to specify a local path, in which the repository
+  # containing the manifest file must be placed.
```

Comment:
This sentence is hard to understand; is there a typo here somewhere? Also, there is a "- " before the "path: project-path" in the below example. That would mean self's value was a list with one element, which is a map. But the schema says self's value is a map, not a list.

It seems like the overall meaning is something like this (with fixed example and suggestion to make path optional applied):

```
The optional path key specifies the local directory to clone this repository into,
in the same way projects can specify their paths. For example, the following would
cause the manifest repository to be checked out into local path "project-path":

self:
  path: project-path

If the path key is missing, the basename of the path component in the manifest repository
URL will be used by default. For example, if the URL is https://example.com/project-repo, the
manifest repository would be cloned to the directory "project-repo".
```

Reply:
Thanks, much better.
Included it (although without `optional`until we've settled on that)